build.gradle

```diff
@@ -67,9 +67,6 @@ configure(project.coreProjects) {
         testCompile ( libraries.powermock_api_mockito)
         testCompile ( libraries.powermock_module_junit4)
         testCompile ( libraries.awaitility)
-
-        jmh "org.openjdk.jmh:jmh-core:1.18"
```

Comment:
JMH should still work. Can you test it?

Reply:
you're right I accidentally broke it.  
Seems like they don't get executed on `gradle check` or `gradle assemble` so I didn't notice. I'll fix this.

Reply:
@RobWin disregard my last comment. I tested it, and it works just fine with the removal, because the plugin already comes with these dependencies.  
You just can't execute `gradle jmh` on the root folder. You have to execute it in a module that acutally has files under `src/jmh`